docs/core_tenets.md

```diff
@@ -0,0 +1,18 @@
+## Jujutsu's Core Tenets
+
+Jujutsu's core tenets are:
+
+ * User-friendliness: Making the  working copy a commit is simpler. This is
+ how the project started.
+ * The "repository", so the commit graph is the source of truth. The working
+ copy is just one way of editing commits.
+ * All operations must be able to scale to Google-scale repos (lots of commits
+ , lots of files): Laziness is important, must avoid accessing data
+ unnecessarily.
+ * Having as few states as possible.
+ * Make it incredibily hard to lose work in your repository.
```

Comment:
`rm -rf .jj` is still going to be easy. Perhaps "Try hard to not lose the user's work"?

Reply:
We already then toned it down here https://github.com/jj-vcs/jj/pull/3089#discussion_r1590138321. I think both work